src/components/Container/Container.tsx

```diff
@@ -2,8 +2,8 @@ import React, { FunctionComponent } from "react";
 
 export const Container: FunctionComponent = ({ children }) => {
   return (
-    <div className="bg-blue-300 min-h-screen">
-      <div className="mx-auto">{children}</div>
-    </div>
+    <main className="bg-blue-300">
+      <div className="container">{children}</div>
+    </main>
```

Comment:
how come change to <main>? im afraid some component is wrapped in a <article>, <aside>, <footer>, <header>, or <nav> then the main got problem liao.

Reply:
right... let me look into this more